app/controllers/merge_requests_controller.rb

```diff
@@ -49,6 +56,35 @@ def remove_merging_organisation
 
 private
 
+  def page
+    params.dig(:merge_request, :page)
+  end
+
+  def next_page_path
+    case page
+    when "absorbing_organisation"
+      if create_new_org?
+        new_org_name_merge_request_path(@merge_request)
+      else
+        confirm_telephone_number_merge_request_path(@merge_request)
+      end
+    when "organisations"
+      absorbing_organisation_merge_request_path(@merge_request)
+    end
+  end
+
+  def previous_template
+    page
+  end
+
+  def redirect_to_next_page
```

Comment:
Shouldn't we save the fact that it's a new organisation in the merge request? For example, if we navigate back if one of the orgs is selected that persists, but if it's new org it clears the answer

Reply:
Would also reduce the need for handling different post methods differently

Reply:
Oh sorry, so on the question`/merge-request/:id/absorbing-organisation` if we select `These organisations are merging into a new one` it doesn't save it, so if we navigate back to this page, nothing is selected.

Also, on the same page, if we select an organisation, but then change it to `These organisations are merging into a new one` that won't be reflected either. If we navigate back to absorbing organisation page it had the previously chosen organisation selected, I assume that's because we don't save this answer `These organisations are merging into a new one`?